circle.yml

```diff
@@ -18,7 +19,7 @@ test:
     - go tool vet *.go                                                   # Go package in root directory.
     - for d in */; do echo $d; done | grep -v tests/ | grep -v third_party/ | xargs go tool vet # All subdirectories except "tests", "third_party".
     - >
-      gopherjs test --short --minify
+      ulimit -s $GOPHERJS_STACK_SIZE && ulimit -s && gopherjs test --short --minify
```

Comment:
Per https://github.com/nodejs/node/issues/14567#issuecomment-319367412, I've not unified the value of `ulimit -s` and the `--stack_size=` flag passed to node.